### DISCUSSION TITLE
How do I develop the vscode setting editor

### DISCUSSION THREAD

**Discussion by fecommunity** [microsoft/vscode-discussions#1229] [Extension Development QnA]
Currently, I am undertaking secondary custom development based on the open-source version of vscode and endeavoring to implement a setting center akin to that of vscode. How can I conduct corresponding secondary development without encroaching upon the existing code? Are there any recommended tutorials and documentation resources available for this purpose?

**Comment by isidorn** [comment]
VS Code is open source, you can look at the source code.
I am closing this question, as this community is for extension authors.
